src/Person.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+
+import './Person.scss';
+
+const Person = ({ name, age, sex, isMarried, partnerName }) => {
+  const status = (!isMarried)
+    ? 'I am not married'
+    : (sex === 'm') ? `My wife's name is ${partnerName}`
+      : `My husband's name is Maksym`;
```

Comment:
Don't create nested ternary operators
Create different variables for 'wife'/'husband' value and full string of marriage status